### ISSUE TITLE
Add `POST`, `GET`, etc. convenience names for `SendPOSTRequest`, etc.

### ISSUE THREAD

**Issue by perrygoy** [ScreenPyHQ/screenpy_requests#1]
All the `Send*Request` actions are nice, but it fits the lingo of API testing better to just say `POST` or `GET` instead. Like this:

```python
the_actor.attempts_to(
    POST.to("https://screenpy.example.com/forexample/").with_(data={"some": "example data"})
)
```

Plus they're much shorter!

**Comment by bandophahita** [comment]
It might make sense to have the POST/GET versions be the original where the others are the convenience names.

**Comment by perrygoy** [comment]
Addressed in #2!
